x/auth/mock/simulate_block.go

```diff
@@ -45,12 +45,12 @@ func GenTx(msg sdk.Msg, accnums []int64, seq []int64, priv ...crypto.PrivKeyEd25
 	for i, p := range priv {
 		sigs[i] = auth.StdSignature{
 			PubKey:        p.PubKey(),
-			Signature:     p.Sign(auth.StdSignBytes(chainID, accnums, seq, fee, msg)),
+			Signature:     p.Sign(auth.StdSignBytes(chainID, accnums, seq, fee, msg, "")),
```

Comment:
Should this ideally just be checking on an empty memo? I think we may want to include some non-empty message, just to check that memo inclusion doesn't break stuff.

Reply:
Changed to include a non-empty message.